### PR TITLE
Lock keg ownership for 2 seconds before explosion

### PR DESCRIPTION
**Pull request by bunniewormy**
## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

Resolves https://github.com/transhumandesign/kag-base/issues/835

### CODE REVIEW

**Review by AsuMagic — COMMENTED** [review]
Should work like that, but I think it would be better if this was implemented in keg scripts instead. i.e. copy over `onAttach` to keg scripts and remove `SetDamageToCarrier.as` from the script list in the keg config file.

File: Entities/Items/Explosives/Keg.cfg [hunk 1]
```diff
@@ -73,8 +73,7 @@ $name                                             = keg
 													ExplodeOnDie.as;
 													NoPlayerCollision.as;
 													BehindWhenAttached.as;
-													SetTeamToCarrier.as;
-													SetDamageToCarrier.as;
+													SetTeamToCarrier.as
```
Comment (eps0003) [inline Entities/Items/Explosives/Keg.cfg:76]: I may be wrong, but doesn't there need to be a semicolon after each script?

Comment (eps0003) [reply Entities/Items/Explosives/Keg.cfg]: I am wrong.

Comment (bunniewormy) [reply Entities/Items/Explosives/Keg.cfg]: not needed, but changed for consistency

**Pull request merged**